Branch-TestBed/Branch-SDK Load Tests/Branch_SDK_Load_Tests.m

```diff
@@ -0,0 +1,78 @@
+//
+//  Branch_SDK_Load_Tests.m
+//  Branch-TestBed
+//
+//  Created by Qinwei Gong on 2/23/15.
+//  Copyright (c) 2015 Branch Metrics. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import "Branch.h"
+#import "BNCPreferenceHelper.h"
+#import "BNCServerInterface.h"
+#import "Nocilla.h"
+
+@interface Branch_SDK_Load_Tests : XCTestCase {
+    
+@private
+    __weak Branch *branch;
+}
+
+@end
+
+@implementation Branch_SDK_Load_Tests
+
+- (void)setUp {
+    [super setUp];
+    
+    branch = [Branch getInstance:@"5668720416392049"];
+    
+    [BNCPreferenceHelper setSessionID:@"97141055400444225"];
+    [BNCPreferenceHelper setDeviceFingerprintID:@"94938498586381084"];
+    [BNCPreferenceHelper setIdentityID:@"95765863201768032"];
+    [BNCPreferenceHelper setLinkClickID:NO_STRING_VALUE];
+    [BNCPreferenceHelper setLinkClickIdentifier:NO_STRING_VALUE];
+    [BNCPreferenceHelper setSessionParams:NO_STRING_VALUE];
+    
+    [branch simulateInitFinished];
+    
+    [[LSNocilla sharedInstance] start];
+}
+
+- (void)tearDown {
+    [[LSNocilla sharedInstance] clearStubs];
+    [[LSNocilla sharedInstance] stop];
+    
+    [super tearDown];
+}
+
+- (void)testLoad {
+    NSDictionary *responseDict = @{@"url": @"https://bnc.lt/l/3PxZVFU-BK"};
+    NSData *responseData = [BNCServerInterface encodePostParams:responseDict];
+    
+    stubRequest(@"POST", [BNCPreferenceHelper getAPIURL:@"url"])
+    .andReturn(200)
+    .withHeaders(@{@"application/json": @"Content-Type"})
+    .withBody(responseData);
+    
+    XCTestExpectation *getShortURLExpectation = [self expectationWithDescription:@"Test getShortURL"];
+    
+    for (int i = 0; i < 1000; i++) {
+        [branch getShortURLWithParams:nil andChannel:[NSString stringWithFormat:@"%d", i] andFeature:nil andCallback:^(NSString *url, NSError *error) {
```

Comment:
Hmm, I'm still not sure there is going to be any value in this test. Even though you're not aiming to test speed, you technically are, because that `waitForExpectations` method will cause the test to fail if it takes more than a second (which, as mentioned, should theoretically never happen, but it's not determinate). Also, what features of the Queue are you testing by filling it with requests; that it can hold 1001 requests? I'm just not sure this test is going to actually prove anything for us.